### DISCUSSION TITLE
ページよってhackmdのみ利用できeditによる編集を禁止できるようにしてほしい。

### DISCUSSION THREAD

**Discussion by kyusuke-10** [growilabs/growi#7411] [Ideas / Requests] [answered]
hackmdで編集していると、他のユーザーがeditでそのページを編集した際に内容が重複したりして変な感じになります。ページ作成者や管理者がhackmdによる編集のみを可能にし、editでの変更を禁止するという設定ページごとにを行えるようになるといいなと思います。
他のユーザーにeditで編集しないように言っても、利用者が多いと間違ってeditで編集するパターンがかなり頻繁にありますので、管理者側で制限したいです。
よろしくお願いいたします。

**Comment by yuki-takei** [comment]
管理画面の Custom CSS で編集ボタンやエディタを隠す(消す)などして HackMD の限定利用に誘導するようなカスタムをしている方はいらっしゃるようです。

ページ毎に、というのは Custom CSS ではできないと思いますのでもしやるとしたら改修ということになると思いますが、本体チームではそのような機能を入れる予定はありません。

**Comment by yuki-takei** [comment, marked as answer]
v7 で HackMD との連携は廃止され、ビルトインエディタで同時多人数編集を実装しました。
本件の想定の使い方がデフォルトになりましたのでクローズします。
